app/src/main/java/com/duckduckgo/app/fire/fireproofwebsite/data/FireproofWebsiteDao.kt

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2020 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.duckduckgo.app.fire.fireproofwebsite.data
+
+import androidx.lifecycle.LiveData
+import androidx.room.*
+
+@Dao
+interface FireproofWebsiteDao {
+
+    @Query("select * from $FIREPROOF_WEBSITES_TABLE_NAME")
```

Comment:
"feels like i updated the value of that constant, everything would still work, though i suspect it would probably break a bunch of things." -> fair point 👍 agree, that it's not something it can be changed without breaking anything.

I wanted to avoid any mistake from my side using the same literal in different classes, but I think your argument is very valid.